package.json

```diff
@@ -25,11 +25,14 @@
     "prepare": "husky install"
   },
   "devDependencies": {
-    "@babel/preset-env": "^7.19.4",
+    "@babel/core": "^7.23.5",
```

Comment:
This @babel/core dependency was necessary but missing, causing yarn install error.